ui/apps/platform/src/Containers/Vulnerabilities/components/GenerateSbomModal.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Alert, Button, Flex, Modal, Text } from '@patternfly/react-core';
+
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+
+export type GenerateSbomModalProps = {
+    onClose: () => void;
+    onGenerateSbom: () => void;
+    imageFullName: string;
+};
+
+function GenerateSbomModal(props: GenerateSbomModalProps) {
+    const { onClose, onGenerateSbom, imageFullName } = props;
+
+    const isLoading = false;
+    const isSuccess = false;
+    const isError = false;
+    const error = undefined;
+
+    function onClickGenerateSbom() {
+        // TODO Implement API call
+        onGenerateSbom();
+    }
+
+    return (
+        <Modal
+            isOpen
+            onClose={onClose}
+            title="Generate Software Bill of Materials (SBOM)"
+            variant="medium"
+            actions={[
+                <Button
+                    key="generate-sbom-action"
+                    isLoading={isLoading}
+                    isDisabled={isLoading}
+                    onClick={onClickGenerateSbom}
+                >
+                    Generate SBOM
+                </Button>,
+                <Button key="close-modal" variant="link" onClick={onClose}>
+                    Cancel
+                </Button>,
+            ]}
+        >
+            <Flex direction={{ default: 'column' }} spaceItems={{ default: 'spaceItemsLg' }}>
+                <Text>
+                    Generate and download the Software Bill of Materials (SBOM) in SBDX 2.3 format.
+                    This file contains a detailed list of all components and dependencies included
+                    in the image.
+                </Text>
+                <Text>
+                    Selected image: <em>{imageFullName}</em>
```

Comment:
PatternFly does not even have a guideline for italic, it is so rare.
What do you think about key-value idiom, either home made or `DescriptionList` elements?
* key has bold font weight
* value has normal font weight

Reply:
This is a good point, and in fact there isn't a single mention of "italic" in the PatternFly documentation. I'll mention this with other UX tweaks I've made during our next SBOM sync.